app/components/github_integration/webhooks/core.py

```diff
@@ -86,6 +90,67 @@ def dict(self, bot: GhosttyBot) -> dict[str, str | None]:
         }
 
 
+class ContentGenerator(Protocol):
+    def __call__(
+        self, event_like: Any, template: str, body: str | None = None, /
+    ) -> EmbedContent: ...
+
+
+class FooterGenerator(Protocol):
+    def __call__(self, event_like: Any, /, *args: Any, **kwargs: Any) -> Footer: ...
+
+
+async def send_edit_difference(
+    bot: GhosttyBot,
+    event: events.IssuesEdited | events.PullRequestEdited,
+    content_generator: ContentGenerator,
+    footer_generator: FooterGenerator,
+) -> None:
+    event_object = (
+        event.issue if isinstance(event, events.IssuesEdited) else event.pull_request
+    )
+
+    if event_object.created_at > dt.datetime.now(tz=dt.UTC) - dt.timedelta(minutes=15):
+        return
+
+    changes = event.changes
+    if changes.body and changes.body.from_:
+        # HACK: replace all 3+ backticks with reverse primes to avoid breaking the diff
+        # block, while maintaining the intent.
```

Comment:
Commas are cool but a pause there reads a bit weird (it feels like the last phrase is hanging off).

```suggestion
        # block while maintaining the intent.
```

(A comma after `reverse primes` doesn't sound odd to me in my opinion, so you could add one there if you wanted to do so.)